npm/pkg/dataplane/policies/chain-management_linux.go

```diff
@@ -123,26 +142,48 @@ func (pMgr *PolicyManager) removeNPMChains() error {
 	return nil
 }
 
-// ReconcileChains periodically creates the jump rule from FORWARD chain to AZURE-NPM chain (if it d.n.e)
-// and makes sure it's after the jumps to KUBE-FORWARD & KUBE-SERVICES chains (if they exist).
-func (pMgr *PolicyManager) ReconcileChains(stopChannel <-chan struct{}) {
-	go pMgr.reconcileChains(stopChannel)
+// reconcile does the following:
+// - cleans up old policy chains
```

Comment:
nitpick: old -> stale
Btw what is d.n.e? May be good to stand for `d.n.e` and use abbreviation.